loader/loader.go

```diff
@@ -205,8 +206,13 @@ func (l *Loader) OpenExistingWallet(pubPassphrase []byte) (w *wallet.Wallet, rer
 	}()
 
 	so := l.stakeOptions
+
+	if so.TicketAddress == nil {
+		return nil, nil
+	}
+
 	w, err = wallet.Open(db, pubPassphrase, so.VotingEnabled, so.AddressReuse,
-		so.TicketAddress, so.PoolAddress, so.PoolFees, so.TicketFee,
+		&so.TicketAddress, &so.PoolAddress, so.PoolFees, so.TicketFee,
```

Comment:
dcrutil.Address is an interface and should be passed by value here, not by pointer.